src/data/location.ts

```diff
@@ -0,0 +1,41 @@
+import * as rt from 'runtypes';
+import { Schedule } from './schedule';
+import { LocationDetail } from './locationDetail';
+
+const Location = rt.Record({
+  kode_lokasi_vaksinasi: rt.Number,
+  nama_lokasi_vaksinasi: rt.String,
+  alamat_lokasi_vaksinasi: rt.String,
+  wilayah: rt.String,
+  kecamatan: rt.String,
+  kelurahan: rt.String,
+  rt: rt.String,
+  rw: rt.String,
+  kodepos: rt.String,
+  jenis_faskes: rt.String,
+  jumlah_tim_vaksinator: rt.Number,
+  nama_faskes: rt.String,
+  created_at: rt.Null,
+  updated_at: rt.Null,
+  pcare: rt.Boolean,
+  jadwal: rt.Array(Schedule),
+  detail_lokasi: rt.Array(LocationDetail),
+  last_updated_at: rt.String
+});
+
+const Locations = rt.Array(Location);
+
+export type ILocation = rt.Static<typeof Location>;
+
+const SCHEDULE_ENDPOINT = `https://vaksin-jakarta.yggdrasil.id/`;
+
+// API CALL
+export async function getVaxLocations() {
+  let response = await fetch(SCHEDULE_ENDPOINT);
+  let data = await response.json();
+  let validateResponse = Locations.validate(data);
+  if (!validateResponse.success) {
+    console.error(`Contract is different from Response`, validateResponse);
```

Comment:
at first I think that I want to throw error here if the contract is different. But since the API will be added or remove very fast. Just log this one into the console.